sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/EventProcessorHost.cs

```diff
@@ -52,10 +52,10 @@ public sealed class EventProcessorHost
 
         /// <summary>
         /// Create a new host to process events from an Event Hub.
-        /// 
+        ///
         /// <para>This overload of the constructor uses the default, built-in lease and checkpoint managers.</para>
         /// </summary>
-        /// <param name="hostName">A name for this event processor host. See method notes.</param>
+        /// <param name="hostName">Name of the processor host. MUST BE UNIQUE. Strongly recommend including a Guid to ensure uniqueness.</param>
```

Comment:
This reads a bit awkwardly to me, and I would prefer that we not emphasize with all capitals.   I'd suggest revising to something like:

`The name to assign the processor host.  This should be a value unique to an EventHub and Consumer Group pairing.  A GUID is recommended.`

To help emphasize, may I also suggest that we add some remarks for additional context.   Perhaps something like:

```csharp
<remarks>
  <para>The <paramref name="hostName" /> is intended to uniquely identify this `EventProcessorHost` instance 
  within the scope of the Event Hub and Consumer Group that it is processing and plays a key part in load balancing.
  If there are two instances with the same <paramref name="hostName" /> processing the same Event Hub and 
  Consumer Group pairing, they will compete for partition ownership and impact the efficiency of load balancing.</para>

  <para>Unless you have a strong reason to do otherwise, it is recommended to assign a <see cref="System.Guid" > as the 
  <paramref name="hostName" />.</para>
</remarks>
```

Reply:
Hey, thanks for the feedback 😄 

I've basically just made sure all the hostName parameters use the wording here: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/EventProcessorHost.cs#L89

Happy to make changes to all of them, as I ran into some trouble after using a constructor signature without the uniqueness warning 😁